docs/release-process.md

```
@@ -162,12 +162,38 @@ Build output expected:
+12. `Bisq-64bit-${NEW_VERSION}.rpm` Red Hat based distro package
+13. `Bisq-64bit-${NEW_VERSION}.rpm.asc` Signature for Red Hat based distro package
+14. `Bisq-64bit-${NEW_VERSION}.exe` Windows installer
+15. `Bisq-64bit-${NEW_VERSION}.exe.asc` Signature for Windows installer
+
+If you need to create and sign the final release directory manually, use the Gradle tasks below. The
+`Bisq-${NEW_VERSION}.jar.txt` file is the concatenation of the four platform jar SHA-256 files produced by
+`./gradlew packageInstallers`; those files are in the VM shared folders / package output directories listed in the
+macOS, Linux, and Windows build sections above. When using `finalize.sh`, the generated file is written to
+`desktop/releases/${NEW_VERSION}/Bisq-${NEW_VERSION}.jar.txt`.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)